### PR TITLE
[PLATFORM-1240] Fix removed join requests not being removed from the list

### PR DESCRIPTION
**Pull request by juhah**
Added fetching the member list after after removing.

### CODE REVIEW

File: app/src/userpages/components/ProductsPage/Members/index.jsx [hunk 1]
```diff
@@ -170,10 +170,14 @@ const Members = () => {
...
             selection.none()
+
+            if (loadMembersRef.current) {
+                loadMembersRef.current()
+            }
         }
 
         setRemoving(false)
-    }, [beneficiaryAddress, remove, selection])
+    }, [beneficiaryAddress, remove, selection, loadMembersRef])
```
Comment (mondoreale) [inline app/src/userpages/components/ProductsPage/Members/index.jsx]: Adding refs a hook dependencies is very optional. They don't change. #fyi

File: app/src/userpages/components/ProductsPage/Members/index.jsx [hunk 2]
```diff
@@ -170,10 +170,14 @@ const Members = () => {
                 })
             }
             selection.none()
+
+            if (loadMembersRef.current) {
+                loadMembersRef.current()
```
Comment (mondoreale) [inline app/src/userpages/components/ProductsPage/Members/index.jsx]: I see! This piece of code pretty much reloads the member list using current filters, etc. Same thing happens on approval (a reload). Slowly getting there…

I guess we're gonna have to add `isMounted()` in both of them callbacks after the `await` loops.

Comment (mondoreale) [reply app/src/userpages/components/ProductsPage/Members/index.jsx]: This change fixes the original issue.

Comment (juhah) [reply app/src/userpages/components/ProductsPage/Members/index.jsx]: Would be better if backend supported deleting multiple entries at the same time but this works for now... let's see how it behaves when we are >100 or even >1000 requests 🙈

Comment (mondoreale) [reply app/src/userpages/components/ProductsPage/Members/index.jsx]: I hear you. I hear you loud and clear.

**Comment by juhah** [comment]
@mondoreale Good point, added the isMounted check. Otherwise, I'll merge this shortly, let's see how it works in real life.

File: app/src/userpages/components/ProductsPage/Members/index.jsx [hunk 3]
```diff
@@ -134,10 +136,10 @@ const Members = () => {
         setApproving(false)
         selection.none()
 
-        if (loadMembersRef.current) {
+        if (isMounted() && loadMembersRef.current) {
```
Comment (mondoreale) [inline app/src/userpages/components/ProductsPage/Members/index.jsx]: I guess we should put `isMounted()` before the above `setApproving` and `selection.none()`.

File: app/src/userpages/components/ProductsPage/Members/index.jsx [hunk 4]
```diff
@@ -170,10 +172,14 @@ const Members = () => {
                 })
             }
             selection.none()
+
+            if (isMounted() && loadMembersRef.current) {
```
Comment (mondoreale) [inline app/src/userpages/components/ProductsPage/Members/index.jsx]: And here, likewise.

File: app/src/userpages/components/ProductsPage/Members/index.jsx (hunk 3 above)
Comment (juhah) [reply app/src/userpages/components/ProductsPage/Members/index.jsx]: Ah, true 👍  Fixed

**Pull request merged**